test/test_basic.py

```diff
@@ -273,15 +273,15 @@ def save(path, im):
     # as they both use freeimage and this has some weird internal formats
     # see https://github.com/scikit-image/scikit-image/issues/1101
     # and https://github.com/imageio/imageio/issues/3
-    from distutils.version import StrictVersion
-    if im.dtype == np.uint16 and StrictVersion(imageio.__version__) <= StrictVersion('0.5.1'):
+    from packaging.version import Version
+    if im.dtype == np.uint16 and Version(imageio.__version__) <= Version('0.5.1'):
         im = im[::-1,::-1]
-    imageio.imsave(path, im)
+    imageio.v3.imwrite(path, im)
 
 def print_stats(rgb):
     # np.min supports axis tuple from 1.10
-    from distutils.version import StrictVersion
-    if StrictVersion(np.__version__) <= StrictVersion('1.10'):
+    from packaging.version import Version
+    if Version(np.__version__) <= Version('1.10'):
```

Comment:
given that we now require `numpy>2`, is this test still needed?

Reply:
Feel free to remove that, it's an ancient version anyway

Reply:
Are you still working on this?

Reply:
Oh, thanks for the ping. I've somehow missed the previous message. let me kick it out :)